models/ipfs_networks.go

```diff
@@ -2,22 +2,23 @@ package models
 
 import (
 	"errors"
+	"time"
 
 	"github.com/RTradeLtd/database/utils"
 	"github.com/jinzhu/gorm"
 	"github.com/lib/pq"
 )
 
 type HostedIPFSPrivateNetwork struct {
-	gorm.Model
-	Name                   string         `gorm:"type:varchar(255)"`
+	Name                   string         `gorm:"primary_key;type:varchar(255)"`
```

Comment:
This might break functionality with our current database setup.
`gorm.Model` includes a few fields `CreatedAt` `UpdatedAt` `DeletedAt` as well as a primary key of type `integer`

Reply:
Also based off our slack conversation, should we change to `gorm:"type:varchar(255);unique"`